src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.targets

```diff
@@ -1,5 +1,5 @@
 <Project>
-    <PropertyGroup Condition="'$(RuntimeIdentifier)' == 'browser-wasm'">
+    <PropertyGroup Condition="'$(RuntimeIdentifier)' == 'browser-wasm' AND '$(UsingBrowserRuntimeWorkload)' == ''">
```

Comment:
We have a weirdo project that targets browser-wasm, does not target the BlazorSDK, but we  really don't want the packs for. I'm able to workaround it by saying `UsingMicrosoftNETSdkBlazorWebAssembly=false`, but that seems a bit iffy.

Reply:
What project/use case is this and why don't you want the packs?

Reply:
It was observed in this case - https://github.com/dotnet/aspnetcore/blob/main/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj. We have a test that simulates trimming for a browser-wasm project. We want to use the browser-wasm runtime, but none of the blazor behavior (since it's not a blazor-app).

Reply:
For a non-blazor *wasm* app, I would expect the Wasm build targets would be needed. Hence, the default.

The particular case of `WasmLinkerTest*` seems to be:
1. `browser-wasm`
2. but doesn't want to use any wasm build bits
- for this very specific case, I think opt-out makes sense.